zrcs/zrc-7.md

```diff
@@ -0,0 +1,202 @@
+
+- `ipfs://QmZIL4tcBsMqLRuCQtPmPe84bpSjrC3Ky7t3JWuHXYB4aA/1`
+- `ipfs://QmZILw65yBXgyfG2ZBg5TrfB2hPjrDQH3RCQFJGkARStAE`
+- `ar://ZILsR4OrYvODj7PD3czIAyNJalub0-vdV_JAg5NqA-o`
+- `https://ipfs.io/ipfs/QmZIL4tcBsMqLRuCQtPmPe84bpSjrC3Ky7t3JWuHXYB4aA/1`
+- `https://foo.mypinata.cloud/ipfs/QmZILYgURKVnLWBm1aXH1BqKqFgmj7j1K6MWAFTkf9xm8A/1`
+- `https://creatures-api.zilliqa.com/api/creature/1`
+
+## II. Abstract
+
+ZRC-7 standardizes the metadata structure and covers token URI optimization with base token URI.
+
+## III. Motivation
+
+The consistent metadata structure can help the NFT creators and builders to handle the NFT metadata more simply. Also, base token URI can be used to reduce the gas cost.
+
+## IV. Specification
+
+### A. Metadata Structure
+
+The metadata must be structured as the following:
+
+| Property            |        Type        | Description                                                                                                                                                                                                                                                                                                                                                    | Required |
+| ------------------- | :----------------: | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | :------: |
+| `name`              |      `String`      | Name of the asset.                                                                                                                                                                                                                                                                                                                                             |    ✓     |
+| `description`       |      `String`      | A human readable description of the asset.                                                                                                                                                                                                                                                                                                                     |          |
+| `resource`          |      `String`      | A URI that points to the asset's resource. A decentralized URI is recommended.                                                                                                                                                                                                                                                                                 |    ✓     |
```

Comment:
This can be either point to centralized storage eg. web server or decentralized file storage eg. IPFS, Arweave